index.html

```diff
@@ -799,14 +799,14 @@ <h4>
     </p>
     <pre class="highlight">
 const reader = new NFCReader();
-reader.scan({ url: "https://mygame.com/mypath/mygame" });
+reader.scan({ id: "https://mygame.com/mypath/mygame" });
 reader.onreading = async event => {
   console.log(`Source:     ${ event.message.url }`);
   console.log(`Game state: ${ JSON.stringify(event.message.records) }`);
 
   const newMessage = {
-    url: "/mypath/mygame/update",
     records: [{
+      id: "/mypath/mygame/update",
```

Comment:
I wonder what the NDEF message id should look like, probably the impl takes this as an input and changes it.
Originally we were giving a path here and the implementation made up the rest.
How to make that clear to devs from the syntax?

Reply:
Documentation should help :-)

- it should be possible to add a absolute URL as long as it matches the origin of the browsing context
- relative urls will be made absolute by us